resources/assets/js/components/Assistant.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+    <div :class="['notification', 'is-radiusless', 'animated', type ? `is-${type}` : '']" v-if="show">
```

Comment:
Can we set 'invisible' in here too, and remove it programmatically once Vue is loaded? Right now if show == false, the browser will still render this for a tick before Vue can hide it which is an annoying quirk of view, but not something we can't overcome.

Reply:
I am new to Vue but are you sure that this is happing this way? I have problem with the page loading but in google cache I found this infos: [Cache](https://webcache.googleusercontent.com/search?q=cache:O8gJtUWNpuEJ:https://forum-archive.vuejs.org/topic/12/v-if-vs-v-show/3+&cd=2&hl=de&ct=clnk&gl=at&client=ubuntu)

Reply:
v-cloak was what I was thinking of; just couldn't remember the name when I made the original comment.

Reply:
Ahh good to know. Thx